src/TraceEvent/Microsoft.Diagnostics.Tracing.TraceEvent.targets

```diff
@@ -0,0 +1,37 @@
+<!--
+***********************************************************************************************
+Microsoft.Bcl.Compression.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
```

Comment:
Note that the previous Nuget package should be viewed as experimental.   It should probably not be used as a template if there are better templates available.    Ideally we start with a completely vanilla nuspec and tweek it just enough to get what we want.   The goals are

1. Simple and straightforward.   
2. A resulting nuget package that current users can pick up very easily.   (Ideally it is compatible, but even that is  negotiable if we can get significant improvements by a modest breaking change).  
3. We wire into existing infrastructure to get the package signed and or released.   

The thing that makes this package a bit harder than normal is that it has native components.    The solution in the past was to include all architectures for the native components in the one package, but we may wish to change to the mechanism used by other packages where you refer to one of several architecture-specific packages.

Our goal is to NOT be innovative, but simply follow a standard for doing what we need (access native components).  

We may wish to split the package (symbol resolution, collection and processing are the obvious groupings).   However I think we can defer that issue for now (for better or worse the are currently together and there is no reason to force this transition now).